### PR TITLE
Add an option to always show the virtual keyboard (special keys)

### PR DESCRIPTION
**Pull request by alescdb**
#159

If someone can check my two resources strings to see if it is correct english or something that looks like english but is not :-)

```
<string name="pref_alwaysvisible_title">Keyboard always visible</string>
<string name="pref_alwaysvisible_summary">Always show the special keys</string>
```

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -644,4 +643,6 @@
 	<!-- Text for button which does not discard changes and keeps editing. -->
 	<string name="discard_host_cancel_button">Keep editing</string>
 
+	<string name="pref_alwaysvisible_title">Keyboard always visible</string>
```
Comment (jklein24) [inline app/src/main/res/values/strings.xml]: nit: I'd rather just have this say "Special keys always visible." not sure we even need the description. with "keyboard" it's a little ambiguous whether you mean the soft keyboard or special keys.

**Comment by alescdb** [comment]
@jklein24 I've made the changes according to your suggestions.

File: app/src/main/java/org/connectbot/ConsoleActivity.java [hunk 2]
```diff
@@ -87,6 +74,20 @@
...
+import org.connectbot.service.TerminalBridge;
+import org.connectbot.service.TerminalKeyListener;
+import org.connectbot.service.TerminalManager;
+import org.connectbot.util.PreferenceConstants;
+import org.connectbot.util.TerminalViewPager;
+
+import java.lang.ref.WeakReference;
+import java.util.ArrayList;
+import java.util.List;
+
```
Comment (kruton) [inline app/src/main/java/org/connectbot/ConsoleActivity.java]: Any reason to rearrange these?

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -644,4 +643,5 @@
 	<!-- Text for button which does not discard changes and keeps editing. -->
 	<string name="discard_host_cancel_button">Keep editing</string>
 
+	<string name="pref_alwaysvisible_title">Special keys always visible</string>
```
Comment (kruton) [inline app/src/main/res/values/strings.xml]: You need to have a comment above to provide context for translators.

File: app/src/main/java/org/connectbot/ConsoleActivity.java (hunk 2 above)
Comment (alescdb) [inline app/src/main/java/org/connectbot/ConsoleActivity.java]: Corrected, sorry about that.
Everytime I touch code and/or xml in ConnectBot I'm struggling with Android Studio and my default settings. Indent with spaces is systematically selected for me in **all** code style configs and imports are automatically rearrange and optimized.

**Pull request merged**